### PR TITLE
Stable mutation toxins

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: You can now stabilize unstable mutation toxin with different chems to get the race you want.
/:cl:

Recipes:

Add these to unstable mutation toxin. These were all obtainable already.
- Human: blood
- Lizard: radium
- Flyperson: mutagen
- Golem: silver
- Podperson: EZ nutrient
- Jelly: slime jelly
- Abductor: morphine (doesn't make much sense but it's all i could think of)
- Android: teslium

These need ADVANCED mutation toxin (from black slimes) instead of unstable.
- Skeleton: milk
- Zombie: toxin
- Ash Lizard: advanced mut. toxin + lizard mut. toxin + ash
- Shadow: liquid dark matter (stabilized ofc) + holy water
- Plasmaman: skeleton mutation toxin + plasma + uranium (if you can shoot advanced mutation toxin into people they're dead already so this should not be too unbalanced, plus it's pretty expensive)

Also added a recipe for the mulligan toxin, with human mutation toxin + mutagen.

Idea from https://tgstation13.org/phpBB/viewtopic.php?f=9&t=7953#p213479

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -392,6 +392,115 @@
...
+
+		var/current_species = H.dna.species.type
+		var/datum/species/mutation = race
+		if(mutation && mutation != current_species)
+			H << mutationtext
+			H.set_species(mutation)
+		else
+			H << "<span class='danger'>The pain vanishes suddenly. You feel no different.</span>"
+
+	return 1
```
Comment (phil235) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: copypasta, maybe you could make unstableslimetoxin a child of stable (or the other way around, whichever is better)

**Comment by PKPenguin321** [comment]
feels like plasmaman is too difficult and skeletons are too easy, although i like the whole "skeletons = milk" gag. maybe make skeleton require krokodil as well or something?

**Comment by XDTM** [comment]
The reason plasmaman is difficult is that if you shoot someone with it they're likely gonna burn and suffocate to death in a few seconds, unless they are carefully prepared.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: i could also make the normal slime toxin a child of stable, it seems its transformation is currently hardcoded into the human species_types

**Comment by optimumtact** [comment]
Skeletons are OP

**Comment by XDTM** [comment]
Nerfed some recipes to need black slimes, and now plasmaman requires milk for the skeleton toxin and uranium, on top of black slime toxin.

**Comment by TechnoAlchemisto** [comment]
Admittedly this requires many many steps in order to succeed in so I doubt it will be abused

**Comment by AdamElTablawy** [comment]
zombie ain't the full on infection zombie, is it

**Comment by PKPenguin321** [comment]
no, zombie as in the infectious one and zombie as in the mutant race are different things

**Comment by LaharlMontogmmery** [comment]
> Plasmaman injection

This is VERY powerfull, as it is pretty much a "Remove from game" syringue - can't clone plasmamen, they immediately catch fire. You'd have to through some hoops to actually clone a plasmaman, and not all geneticists are going to bother. 

Requirement are kind of steep/awkward to acquire (milk + black slime + mining doing their job), however. So I think we can let this slide.

It's going to get hilarious if someone manages, tho.

**Comment by XDTM** [comment]
Shadowman further nerfed by requiring holy water.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -360,38 +360,147 @@
 	description = "A corruptive toxin produced by slimes."
 	color = "#13BC5E" // rgb: 19, 188, 94
 
-/datum/reagent/unstableslimetoxin
-	name = "Unstable Mutation Toxin"
-	id = "unstablemutationtoxin"
-	description = "An unstable and unpredictable corruptive toxin produced by slimes."
+/datum/reagent/stableslimetoxin
+	name = "Human Mutation Toxin"
+	id = "humanmutationtoxin"
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: change this to "stableslimetoxin"

Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Done

**Comment by Anonmare** [comment]
👍 
Been suggesting this for a while tbh.

**Comment by Jazaen** [comment]
Maybe you could limit it to just roundstart+ with normal toxin?
Also, abductor-change could be used to cheese game- or admin-spawned ones.

**Comment by XDTM** [comment]
Abductor is already obtainable through unstable toxin

**Comment by Anonmare** [comment]
@Jazaen  I don't think you can, I've never seen xenobio abductors being able to hear an abductor team before.

**Comment by Cheridan** [comment]
sure why not

**Comment by Iamgoofball** [comment]
i cant wait for the first perma for "unconsensual transformation roleplay"

**Comment by Jazaen** [comment]
@Anonmare Huh, I thought it was racial. I'm on mobile atm, so I can't really check code.

@Iamgoofball Nothing forced braintransplants couldn't do! This one just requires way less work abducting people or building telesci.

**Comment by XDTM** [comment]
Don't forget the lings transforming everyone into lizards

**Comment by Jazaen** [comment]
Those are antags material, and thus validsalad.
UNLESS you make them from xenobio. !!SCIENCE!!

**Comment by optimumtact** [comment]
Lets see how we go

**Pull request merged**